### ISSUE TITLE
feat: ELI5 tooltips for icons and numbers

### ISSUE THREAD

**Issue by r0man1337** [BibliothecaForAdventurers/realms-react#373]
We have a lot of icons and numbers without description/explanation across the project.

Need to go through all the popups and pages, and add tooltips with a brief explanation, so that a person who sees the game for the first time can quickly understand what's going on.

Few examples of unclear things:
![image](https://user-images.githubusercontent.com/4090500/214835762-c1f322b3-7c1e-407e-8614-f8d1b7e1b991.png)
![image2](https://user-images.githubusercontent.com/4090500/214835770-4cc92764-3978-4203-9036-38fe37bbbe6c.png)

**Comment by ponderingdemocritus** [comment]
added in #391
